spec.emu

```diff
@@ -882,6 +957,217 @@ contributors: Google, Ecma International
+      <h1>
+        CreateMonthCode (
+          _number_: an integer,
+          _isLeap_: a Boolean,
+        ): a String
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It creates a month code with the given month number and leap month flag.
+        </dd>
+      </dl>
+      <p>It performs the following steps when called:</p>
+      <emu-alg>
+        1. Let _numberPart_ be ToZeroPaddedDecimalString(_number_, 2).
+        1. If _isLeap_ is *true*, then
+          1. Return the string-concatenation of the code unit 0x004D (LATIN CAPITAL LETTER M), _numberPart_, and the code unit 0x004C (LATIN CAPITAL LETTER L).
+        1. Else,
+          1. Return the string-concatenation of the code unit 0x004D (LATIN CAPITAL LETTER M) and _numberPart_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-calendardatearithmeticaltoiso" type="implementation-defined abstract operation">
+      <h1>
+        CalendarDateArithmeticalToISO (
+          _calendar_: a calendar type that is not *"iso8601"*,
+          _year_: an integer,
+          _arithmeticalMonth_: a positive integer,
+          _day_: an integer,
+        ): either a normal completion containing an ISO Date Record or a throw completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It returns an ISO Date Record that, when converted to a Calendar Date Record with _calendar_ (for example, with CalendarISOToDate), contains the given _year_, _arithmeticalMonth_, and _day_ values in its [[Year]], [[Month]], and [[Day]] fields. If this is not possible, a RangeError is thrown.
+        </dd>
+      </dl>
+      <p>It performs the following steps when called:</p>
+      <emu-alg>
+        1. If _year_, _arithmeticalMonth_, and _day_ do not form a valid date in _calendar_, throw a RangeError exception.
+        1. Return a calendar-dependent ISO Date Record as described above.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-calendardaysinmonth" type="implementation-defined abstract operation">
+      <h1>
+        CalendarDaysInMonth (
+          _calendar_: a calendar type that is not *"iso8601"*,
+          _year_: an integer,
+          _arithmeticalMonth_: a positive integer,
+        ): an integer
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It interprets _year_ and _arithmeticalMonth_ as arithmetical values in the given _calendar_ and returns the number of days in the corresponding month.
+        </dd>
+      </dl>
+      <p>It performs the following steps when called:</p>
+      <emu-alg>
+        1. Return a calendar-dependent integer as described above.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-calendarmonthsinyear" type="implementation-defined abstract operation">
+      <h1>
+        CalendarMonthsInYear (
+          _calendar_: a calendar type that is not *"iso8601"*,
```

Comment:
Is there a reason why we chose "calendar type" instead of "calendar identifier" to align with "time zone identifier" ?

Reply:
It was the term already in use in 402.